pkg/performanceprofile/cmd/render/render.go

```diff
@@ -191,29 +189,12 @@ func render(inputDir, outputDir string) error {
 			return err
 		}
 
-		uid := pp.UID
-		if uid == types.UID("") {
-			uid = uuid.NewUUID()
-		}
-
-		or := []v1.OwnerReference{
-			{
-				Kind:       pp.Kind,
-				Name:       pp.Name,
-				APIVersion: pp.APIVersion,
-				UID:        uid,
-			},
-		}
-
 		for _, componentObj := range components.ToObjects() {
-			componentObj.SetOwnerReferences(or)
+			componentObj.SetOwnerReferences(nil)
```

Comment:
Ideally this logic and this commit (which I think is getting ready to merge) should be split to its own PR because it's a related but different change wrt to "add default MCP objects for rendering"

Reply:
Sounds good, updated